.github/workflows/update_dashboard.yml

```
@@ -7,6 +7,7 @@ on:
     paths:
       - 'dff/utils/docker/dockerfile_stats'
       - 'dff/utils/docker/entrypoint_stats.sh'
```

With the addition of other files:
```suggestion
      - 'dff/utils/docker/**'
```